src/test/looping_control_test.cpp

```diff
@@ -322,31 +322,47 @@ TEST_F(LoopingControlTest, LoopOutButton_AdjustLoopOutPointInsideLoop) {
     EXPECT_FRAMEPOS_EQ_CONTROL(mixxx::audio::FramePos{1500}, m_pLoopEndPoint);
 }
 
-TEST_F(LoopingControlTest, LoopInOutButtons_QuantizeEnabled) {
+TEST_F(LoopingControlTest, DISABLED_LoopInOutButtons_QuantizeEnabled) {
```

Comment:
can you add a short comment that links to the issue?

Reply:
Yup, done.